src/test/java/org/springframework/samples/petclinic/model/PersonTest.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.samples.petclinic.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class PersonTest {
+
+    private Person personObj;
+
+    @Before
+    public void setUp(){
+        personObj = new Person();
+    }
+
+    // Testing the DAO of Person class by verifying the setter and getter of FirstName
+    @Test
+    public void testSetAndGetFirstName() {
+        personObj.setFirstName("Johnny");
```

Comment:
Is the `getFirstName` supposed to be getting tested here?

Reply:
Added the assertEqual, must have forgotten it.